modules/azure-redis/module.yaml

```
@@ -1,9 +1,11 @@
 name: azure-redis
 sources:
   - https://github.com/Unique-AG/terraform-modules/tree/main/modules/azure-redis
-version: 2.0.0-rc.2
+version: 2.0.0-rc.3
```

here its correct, even tho we could go for 2.0.0 now as we use in prod.
```suggestion
version: 2.0.0
```